src/main/java/spoon/Launcher.java

```diff
@@ -88,6 +90,7 @@
 	protected JSAPResult jsapActualArgs;
 
 	private List<String> processors = new ArrayList<String>();
+	private List<Processor<? extends CtElement>> processorsInstance = new ArrayList<Processor<? extends CtElement>>();
```

Comment:
oh call the processors field processorTypes to math the getter method instead of processorNames